data/crac/crac-api/src/main/java/com/farao_community/farao/data/crac_api/ElementaryAction.java

```diff
@@ -0,0 +1,15 @@
+package com.farao_community.farao.data.crac_api;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.powsybl.iidm.network.Network;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public interface ElementaryAction {
+    /**
+     * Trigger the actions on a given network.
```

Comment:
Apply instead of trigger ? Trigger feels more like a contingency

Reply:
ok